GitPractice.Domain/StudentsProvider.cs

```diff
@@ -15,6 +15,7 @@ public StudentsProvider()
                 { new GroupNumber("IC-32"), new[]{new Student("FirstName", "LastName") }.ToImmutableList() },
                 { new GroupNumber("IC-33"), new[]{new Student("FirstName", "LastName") }.ToImmutableList() },
                 { new GroupNumber("IC-34"), new[]{new Student("FirstName", "LastName") }.ToImmutableList() },
+                { new GroupNumber("IC-31"), new[]{new Student("Kateryna", "Soloviova") }.ToImmutableList() },
```

Comment:
Use one group list for all group students

Reply:
done